stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md

```diff
@@ -0,0 +1,65 @@
+# Naive-Bayes-Classifier-Flux
+Implementation of Naive Bayes Classifier in Flux
+
+Awesome! So you want to get started with ML in Flux? Let's dive in! 
+
+Steps: 
+
+Prerequesites: InfluxDB 2.0 (local or Cloud instance), this repo (includes zoo-data.csv, script.py and naiveBayesClassifier.flux) 
+InfluxDB: https://github.com/influxdata/influxdb
+
+1) Ensure you have InfluxDB 2.0 set up by either going to localhost:9999 (if you `make` InfluxDB from source) or going to your Cloud instance on AWS, Azure, etc. Keep the instance running throughout this demo.  
+2) Choose a binary dataset (fields can only take on 2 unique values) or visit https://archive.ics.uci.edu/ml/datasets.php for some wonderful datasets you can get started with. 
+
+![](images/csvData.png)
+
+3) You will have to make edits to script.py in the following areas: 
+
+- Set `mydata` equal to the file path of your dataset 
+- Fill in your values for `token`, `bucket`, and `organization`. (If you are using my sample zoo dataset you can ignore the rest of this step) 
+- Ensure that `periods` is set equal to the value you get from running the `data.size` cell. 
+- Set `dataframe_measurement_name` to be the name of your dataset. 
+- Note which fields you want to use as actual fields you classify on and which you would like to use for Class. Recall we predict `P(Class | field)`.
+- Anything you choose to be a class must be listed inside of `data_frame_tag_columns` and everything else defaults to a `field`. Finally, run the script. You've just written your dataset to an InfluxDB bucket. 
+
+![](images/pythonScript.png)
```

Comment:
@RohanSreerama5 Nathaniel pointed out that we don't typically added images to READMEs with data that can be pasted in as text. Would you mind removing the images and either adding this data as text or linking to the existing csvs in this project that contain the data?